README.rst

```diff
@@ -32,24 +32,28 @@ Running Tests
 
 You can run all the tests by executing:
 
-    python -m venv venv
+    python -m venv myvirtualenvironmentname
```

Comment:
venv is fine here, most Pythyon tutorials use 'venv' so that's consistent with other packages and Python tutorials.